data/movefree

```diff
@@ -0,0 +1,6 @@
+keepmovingwithmovefree.com
+move-free.net
+movefree.cn
+movefrees.com
```

Comment:
[movefree.com](movefree.com) redirects to [Schiff Vitamins](https://www.schiffvitamins.com/pages/move-free). Would be nice to include that.

Reply:
Fixed.